doc/apimanager-reference.md

```diff
@@ -152,6 +152,8 @@ Only one of the fields can be chosen. If no field is specified then PVC is used.
 | **Field** | **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
 | --- | --- | --- | --- | --- | --- |
 | StorageClassName | `storageClassName` | string | No | nil | The Storage Class to be used by the PVC |
+| Resources | `resources` | [PersistentVolumeClaimResourcesSpec](apimanagerbackup-reference.md#PersistentVolumeClaimResourcesSpec) | No | nil | The minimum resources the volume should have |
```

Comment:
Referencing APIManagerBackup seems a little bit weird. Descriptions there are in the context of backup too. For example in the linked url "Size of the PersistentVolumeClaim where the backup is to be performed. Set enough size to contain all data that is backed up."

Can we duplicate the content here and adapt it? Or maybe linking to the K8s resources API type documentation and describing that is limited to some specific fields inside of it

Reply:
You mean to duplicate also the [type](https://github.com/3scale/3scale-operator/blob/ec9187a45d0de68f2c943e73940a09310bd450f3/pkg/apis/apps/v1alpha1/apimanagerbackup_types.go#L53) 
```
type PersistentVolumeClaimResources struct {
	// Storage Resource requests to be used on the PersistentVolumeClaim.
	// To learn more about resource requests see:
	// https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
	Requests resource.Quantity `json:"requests"` // Should this be a string or a resoure.Quantity? it seems it is serialized as a string
}
```

Or maybe take it to some `commons` and reuse from any CRD.

Reply:
I meant at documentation level only

Reply:
:+1: